addons/website_sale/data/demo.xml

```diff
@@ -63,7 +63,7 @@
 
         <record id="product.product_product_6" model="product.product">
             <field name="is_published" eval="True"/>
-            <field name="website_sequence">10010</field>
+            <field name="website_sequence">10200</field>
```

Comment:
Wouldn't it be better to add the missing `website_sequence` to all the demo data's products so that they are actually unique, preventing the ORM to batch setting them the default of 10000?

Reply:
Some products are defined in the `product` module, and `website_sequence` is introduced in the `website_sale` module, so the field won't exist in the parent model.